inst/rmarkdown/templates/tutorial/resources/tutorial-format.js

```diff
@@ -153,6 +153,10 @@ $(document).ready(function() {
 
     }
 
+    function i18nextLang(fallbackLng = 'en') {
```

Comment:
Need to make this es5 like. It will be safe guarded by the `|| 'en'`

```suggestion
    function i18nextLang(fallbackLng) {
```

Reply:
Oh right, good catch, thanks!